app/src/main/java/ca/rmen/android/poetassistant/Tts.java

```diff
@@ -128,30 +146,36 @@ public void onInit(int status) {
             }
             EventBus.getDefault().post(new OnTtsInitialized(status));
         }
-    }
+    };
```

Comment:
Since we can init multiple times now (thanks to the new public `restart()` method), we should just have one instance of a `OnInitListener`, not create an anonymous each time we init.